perses/tests/test_repex.py

```diff
@@ -416,7 +416,7 @@ def test_RESTCapableHybridTopologyFactory_repex_charge_transformation():
                 # Extract uncorrelated energy matrix (u_ln) and samples from states (N_l)
                 energy_matrix, samples_per_state = analyzer._compute_mbar_decorrelated_energies()
                 # Compute free energies with boostrapping using pymbar
-                mbar = pymbar.MBAR(energy_matrix, samples_per_state, nbootstraps=200)
+                mbar = _pymbar_bar(energy_matrix, samples_per_state, nbootstraps=200)
```

Comment:
Wait, this is MBAR, I think you had it right from the beginning, this hasn't change in pymbar 4 so we should be using it as we had it before without problems! Apologies for the confusion,